src/com/sahokia/kos/concerttickets/model/User.java

```diff
@@ -0,0 +1,14 @@
+package com.sahokia.kos.concerttickets.model;
+
+import java.util.UUID;
+
+public abstract class User extends Identifiable {
+    User() {
+        super.setId(UUID.randomUUID().toString());
```

Comment:
usually we don't use setters/getters in constructor -> we have direct access to fields

Reply:
Applied direct acces to the fields instead of setters in contructors